go/install/fuse_status_windows.go

```diff
@@ -51,6 +54,52 @@ func detectDokanDll(dokanPath string, log Log) bool {
 	return exists
 }
 
+// Read all the uninstall subkeys and find the ones with DisplayName starting with "Dokan Library"
+// and containing "Bundle"
+func findDokanUninstall(wow64 bool) (result string) {
+	var access uint32 = registry.ENUMERATE_SUB_KEYS | registry.QUERY_VALUE
+	// Assume this is build 32 bit, so we need this flag to see 64 bit registry
+	//   https://msdn.microsoft.com/en-us/library/windows/desktop/aa384129(v=vs.110).aspx
+	if wow64 {
+		access = access | registry.WOW64_64KEY
+	}
+
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall", access)
+	if err != nil {
+		fmt.Printf("Error %s opening uninstall subkeys\n", err.Error())
+		return
+	}
+	defer k.Close()
+
+	names, err := k.ReadSubKeyNames(-1)
+	if err != nil {
+		fmt.Printf("Error %s reading subkeys\n", err.Error())
+		return
+	}
+	for _, name := range names {
+		subKey, err := registry.OpenKey(k, name, registry.QUERY_VALUE)
+		if err != nil {
+			fmt.Printf("Error %s opening subkey %s\n", err.Error(), name)
+		}
+
+		displayName, _, err := subKey.GetStringValue("DisplayName")
+		if err == nil && strings.HasPrefix(displayName, "Dokan Library") && strings.Contains(displayName, "Bundle") {
```

Comment:
Are we swallowing the error here? Would be clearer if it was:
```go
// Some package scoped region above:
dokanRegexp := regexp.MustCompile("^Dokan Library.*Bundle")
...
if err != nil {
    fmt.Printf("...")
    continue
}
if !dokanRegexp.MatchString(displayName) {
    continue
}
```

Reply:
The error isn't worth logging